### PR TITLE
USHIFT-437: Opt into automated branching

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: ci-operator/config/openshift/microshift/openshift-microshift-main.yaml [hunk 1]
```diff
@@ -60,6 +60,12 @@ images:
       - destination_dir: .
         source_path: /usr/bin/openshift-tests
   to: dependency-payload
+promotion:
+  excluded_images:
+  - dependency-payload
+  - microshift-validate
+  name: "4.12"
```
Comment (dhellmann) [inline ci-operator/config/openshift/microshift/openshift-microshift-main.yaml:67]: Do we want this to say 4.13 now?

**Review by pacevedom — APPROVED** [review]
/lgtm

**Pull request merged**